core/src/test/scala/org/broadinstitute/dsde/rawls/entities/local/LocalEntityProviderSpec.scala

```diff
@@ -537,7 +538,9 @@ class LocalEntityProviderSpec extends AnyWordSpecLike with Matchers with ScalaFu
     "insert cache record when updating if non-existent" in withLocalEntityProviderTestDatabase { _ =>
       val wsid = localEntityProviderTestData.workspace.workspaceIdAsUUID
       val workspaceFilter = entityCacheQuery.filter(_.workspaceId === wsid)
-      val expectedTimestamp = Timestamp.from(Instant.now())
+      // Java Instants can have greater precision than java.sql.Timestamps read from the db,
+      // so make sure we truncate the Instant here
+      val expectedTimestamp = Timestamp.from(Instant.now().truncatedTo(ChronoUnit.MILLIS))
```

Comment:
see https://broadinstitute.slack.com/archives/C03B4RKTA5B/p1650028821220959 for more discussion